### DISCUSSION TITLE
Can i run nest js on render.com

### DISCUSSION THREAD

**Discussion by AgrichubAdmin** [community/community#76597] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Its like am failing to run a nest application on render.com

**Comment by danielmarv** [comment, marked as answer]
Hey @AgrichubAdmin 
Yap it is possible because i have ran quite a lot of staffs
but its better when you use docker to run the nest js application
Ref: https://amplication.com

**Reply by AgrichubAdmin** [reply]
thanx @danielmarv

**Reply by ghostinhershell** [reply]
Thanks for jumping in here @danielmarv! 🎉

**Reply by leeringrose** [reply]
Thanks for sharing! @danielmarv  It would be also quite helpful for me.
